server/controllers/recommendations/createRecommendation.ts

```diff
@@ -2,10 +2,15 @@ import { Request, Response } from 'express'
 import { createRecommendation } from '../../data/makeDecisionApiClient'
 import { validateCrn } from '../../utils/utils'
 import { routeUrls } from '../../routes/routeUrls'
+import { trackEvent } from '../../monitoring/azureAppInsights'
+import { EVENTS } from '../../utils/constants'
+import logger from '../../../logger'
 
 export const createRecommendationController = async (req: Request, res: Response): Promise<Response | void> => {
   const normalizedCrn = validateCrn(req.body.crn)
   try {
+    logger.info(`About to track the MRD_RECOMMENDATION_STARTED event to app insights`)
+    trackEvent(EVENTS.MRD_RECOMMENDATION_STARTED, req)
```

Comment:
I'd move this after line 15 (the redirect). That way, if it throws, the redirect will still happen.